turtlebot_description/urdf/turtlebot_properties.urdf.xacro

```diff
@@ -23,7 +23,7 @@
      (currently in development and awaiting feedback)
    -->
 	<property name="cam_px" value="-0.087" />
-	<property name="cam_py" value="-0.0125" />
+<!--	<property name="cam_py" value="-0.0125" />-->
```

Comment:
Might want to add some explanation here, or link to a pull request. It's a bit obscure for future devs having it just 'commented'.